src/components/form/HdInputPhone.vue

```diff
@@ -0,0 +1,368 @@
+      } else {
+        window.removeEventListener('keyup', selectKeys);
+      }
+    },
+  },
+  methods: {
+    toggleDropdown() {
+      this.showDropdown = !this.showDropdown;
+    },
+    selectCountry(country) {
+      this.selectedCountry = country;
+      this.toggleDropdown();
+      if (this.phoneNumber.length <= MAX_DIGITS_DIAL_CODE) {
+        this.$refs.input.$refs.input.$refs.input.focus();
+      }
+    },
+    phoneFormatter(phone) {
+      return formatPhoneNumber(this.selectedCountry.dial_code, phone);
+    },
+    keysHandler(event) {
+      const { keyCode } = event;
+      const firstCountryCode = this.sortedCountriesList[0].code;
+      const lastCountryCode = this.sortedCountriesList[this.sortedCountriesList.length - 1].code;
+      let countryNode = null;
+
+      const foucsCountryCode = () => {
+        if (countryNode !== null) {
+          countryNode.firstChild.focus();
+          this.focusedCountry = countryNode.id;
+        }
+      };
+
+      if (keyCode === ARROW_DOWN_KEY_CODE) {
+        if (this.focusedCountry === null) { // If there is no focus yet, focus first option
+          [countryNode] = this.$refs[firstCountryCode];
+        } else if (this.focusedCountry !== lastCountryCode) { // If there is, focus next option
+          countryNode = this.$refs[this.focusedCountry][0]?.nextSibling;
+        }
+        foucsCountryCode();
+      } else if (keyCode === ARROW_UP_KEY_CODE) {
+        if (this.focusedCountry === null) { // If there is no focus yet, focus last option
+          [countryNode] = this.$refs[lastCountryCode];
+        } else if (this.focusedCountry !== firstCountryCode) { // If there is, focus prev option
+          countryNode = this.$refs[this.focusedCountry][0]?.previousSibling;
+        }
+        foucsCountryCode();
+      } else if (isAlphabetKey(keyCode)) {
+        const keyString = event.key.toUpperCase();
+        const firstMatchingCountryCode = this.sortedCountriesList.find(
+          (country) => this.t.COUNTRIES[country.code].charAt(0) === keyString,
+        )?.code;
+        if (firstMatchingCountryCode) {
+          [countryNode] = this.$refs[firstMatchingCountryCode];
+          foucsCountryCode();
+        }
+      } else if (keyCode === ESC_KEY_CODE) {
+        this.toggleDropdown();
+      }
+    },
+    handleInputEvent(value) {
+      this.$emit('input', formatPhoneNumber(this.selectedCountry.dial_code, value));
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'homeday-blocks/src/styles/mixins.scss';
+
+$flag-icons-path: '../../../node_modules/flag-icons-svg/svg';
+@import './node_modules/flag-icons-svg/sass/_variables.scss';
+@import './node_modules/flag-icons-svg/sass/flag-icons.scss';
+
+$dropdown-button-width: 74px;
+$dropdown-button-height: 55px;
+
+.phone-input {
+  position: relative;
+  display: flex;
+
+  ::v-deep {
+    .field__main .input {
+      border-top-left-radius: 0 !important;
```

Comment:
![every-time-you-use-important-in-css-1-kitten-dies](https://user-images.githubusercontent.com/14016028/143426936-c2d01186-d9b3-4142-a8fe-540d709cf196.jpg)

I just wanted to make sure that you are informed about this situation. 😄  Avoiding the usage of `!important` is essential, but it is also okay if you don't have any other way to go. :)

Reply:
I know I hate using this one haha I am going to see if I have a workaround, I think that adding more specificity to the selector should work 👍🏼